packages/cra-template-redux-typescript/template/src/index.tsx

```diff
@@ -1,23 +1,23 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
-import { Provider } from 'react-redux';
-import { store } from './app/store';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
-import './index.css';
+import React from "react"
+import { createRoot } from "react-dom/client"
+import { Provider } from "react-redux"
+import { store } from "./app/store"
+import App from "./App"
+import reportWebVitals from "./reportWebVitals"
+import "./index.css"
 
-const container = document.getElementById('root')!;
-const root = createRoot(container);
+const container = document.getElementById("root") as HTMLDivElement
```

Comment:
rather than asserting here, we could honestly check it
```ts
const container = document.getElementById("root");
if (container) {
  const root = createRoot(container)
  root.render(...)
} else {
  throw new Error("no root found!")
}
```
no type assertions necessary